pkg/ottl/contexts/ottllog/README.md

```diff
@@ -11,6 +11,8 @@ The following fields are the exception.
 
 | path                                           | field accessed                                                                       | type                                                                    |
 |------------------------------------------------|--------------------------------------------------------------------------------------|-------------------------------------------------------------------------|
+| ottl.storage                                   | the value of the current transform context's temporary storage                       | pcommon.Map                                                             |
```

Comment:
I am wishing this name wasn't as long; it's gonna be tedious to type `ottl.storage["some_name"]` a lot. Given that it's just a temporary value, and we kind of want to make its temporary nature really clear to people so they don't expect long-term storage beyond the scope of the current context, can we just call it `temp` or even `tmp`?

Reply:
I agree the name came out very long.  I'd support `tmp` as the path name.